src/routes/eventRouter.js

```diff
@@ -23,23 +23,33 @@ routes.post('/event', (request, res) => {
     if (!request.body || request.body == {}){
       return res.send("no request body")
     }
-    var event = new Event({
+    let event = new Event({
         title: request.body.title,
         location: request.body.location,
         startTime: request.body.startTime,
         endTime: request.body.endTime,
         checkIn: request.body.checkIn,
-        url: request.body.url
+        url: request.body.url,
+        tags: request.body.tags
     }).save((err, response) => {
-        if (err) res.status(400).send(err)
-        res.status(200).send(response)
+        if (err) res.status(400).send(err);
+        res.status(200).send(response);
     })
     
 });
 
+routes.delete('/event', (request, res) => {
+    try {
+      Event.findByIdAndDelete(request.body.id);
+      res.status(200).send("deleted");
+    } catch (err) {
+      console.log('error ', err);
+    }
+});
+
```

Comment:
We have merge conflicts for the lines above/already implemented so we could probably take these out